README.md

```diff
@@ -16,12 +16,25 @@ SUBCOMMANDS:
 ```
 
 ### Example Usage
-
+#### Sway
 You can use pw-volume to bind multimedia keys to raise, lower, and mute volume.
 For instance, in Sway's config:
 
 ```
-bindsym XF86AudioRaiseVolume exec pw-volume change +2.5%
-bindsym XF86AudioLowerVolume exec pw-volume change -2.5%
-bindsym XF86AudioMute exec pw-volume mute toggle
-```
\ No newline at end of file
+bindsym XF86AudioRaiseVolume exec "pw-volume change +2.5%; pkill -RTMIN+8 waybar"
+bindsym XF86AudioLowerVolume exec "pw-volume change -2.5%; pkill -RTMIN+8 waybar"
+bindsym XF86AudioMute exec "pw-volume mute toggle; pkill -RTMIN+8 waybar"
+```
+#### Waybar
+```
+"custom/pipewire": {
+    "exec": "pw-pipewire status",
+    "return-type": "json",
+    "interval": once,
+    "signal": 8,
+    "format": "{icon} {percentage}",
+    "format-icons": {
+            "default": ["󰕿", "󰖀", "󰕾"]
+        },
```

Comment:
indentation

Reply:
How should it be?

Reply:
Sorry, I meant there's an extra tab on these two lines.
It should look more like
```
"custom/pipewire": {
    "exec": "pw-pipewire status",
    "return-type": "json",
    "interval": "once",
    "signal": 8,
    "format": "{icon} {percentage}",
    "format-icons": {
        "mute": "",
        "default": ["󰕿", "󰖀", "󰕾"]
    }
},
```